api/v3/openapi.yaml

```diff
@@ -1011,15 +1011,27 @@ components:
       type: object
       required:
         - topic_name
-        - partitions_count
-        - replication_factor
       properties:
         topic_name:
           type: string
         partitions_count:
           type: integer
         replication_factor:
           type: integer
+        replicas_assignments:
+          type: array
+          items:
+            type: object
+            required:
+              - partition
+              - replicas
+            properties:
+              partition:
```

Comment:
Whenever we are talking about an identifier in the REST Proxy API, we are explicit about it. So here you should use `partition_id` instead of `partition`. If you look at the API you will notice that whenever `partition` is used, that is referring to the `Partition` resource, i.e., it will be a link to the partition.

Reply:
Then maybe also change `replicas` to `broker_id_list` or something similar?